client/components/PubShareDialog/PubShareDialog.tsx

```diff
@@ -12,15 +12,18 @@ import {
 import { usePageContext, usePendingChanges } from 'utils/hooks';
 import { useMembersState } from 'client/utils/members/useMembers';
 import { pubUrl } from 'utils/canonicalUrls';
+import { usePubContext } from 'containers/Pub/pubHooks';
 
 require('./pubShareDialog.scss');
 
 type PubShareDialogProps = {
 	isOpen: boolean;
 	onClose: (...args: any[]) => any;
 	pubData: {
```

Comment:
Might be good to just bring the `Pub` type in here (these prop definitions were likely written before it existed.)